doc/manual/src/language/string-context.md

```diff
@@ -0,0 +1,138 @@
+# String context
```

Comment:
This needs a warning telling that it's an advanced thing that people probably don't need to know unless they really do